### PR TITLE
LUN-237: Optimize trophy icons

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]
Tested on android

File: src/components/__tests__/Trophy.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Trophy from '../Trophy'
+
+describe('Trophy', () => {
+  it('should show no trophy when level is 0', () => {
+    const { queryByTestId } = render(<Trophy level={0} />)
+    expect(queryByTestId('trophy-0')).toBeNull()
+  })
+
+  it('should show one trophy when level is 1', () => {
+    const { queryByTestId } = render(<Trophy level={0} />)
```
Comment (Taggotty) [inline src/components/__tests__/Trophy.spec.tsx:13]:
```suggestion
    const { queryByTestId } = render(<Trophy level={1} />)
```

File: src/components/__tests__/Trophy.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    expect(queryByTestId('trophy-0')).toBeDefined()
+    expect(queryByTestId('trophy-1')).toBeNull()
+  })
+
+  it('should show three trophy when level is 3', () => {
+    const { queryByTestId } = render(<Trophy level={0} />)
```
Comment (Taggotty) [inline src/components/__tests__/Trophy.spec.tsx:19]:
```suggestion
  it('should show three trophies when level is 3', () => {
    const { queryByTestId } = render(<Trophy level={3} />)
```

File: src/components/Trophy.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+`
+
+interface PropsType {
+  level: number
+}
+
+const Trophy = ({ level }: PropsType): ReactElement => {
+  const trophies = []
+  for (let i = 0; i < level; i += 1) {
+    trophies.push(<TrophyIconStyled key={`trophy-${i}`} testID={`trophy-${i}`} width={15} height={15} />)
```
Comment (Taggotty) [inline src/components/Trophy.tsx:23]: Just curious. Why didn't you move width and height into the component style specifications above?

**Review by Taggotty — APPROVED** [review]

**Comment by Taggotty** [comment]
Tested on android emulator

File: src/components/__tests__/Trophy.spec.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/components/__tests__/Trophy.spec.tsx:13]: oke null is also defined, therefore the test were not failing and completely useless. Fixed it.

File: src/components/Trophy.tsx (hunk 3 above)
Comment (SteffiStuffel) [reply src/components/Trophy.tsx:23]: Because it did not work, not sure why.

**Pull request merged**